scripts/remove-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# prevent sourcing of this script, only allow execution
+$(return >/dev/null 2>&1)
+test "$?" -eq "0" && { echo "This script should only be executed." >&2; exit 1; }
+
+# exit on errors, undefined variables, ensure errors in pipes are not hidden
+set -Eeuo pipefail
+
+declare mydir
+mydir=$(pwd -P)
```

Comment:
please use the method already in use in the other scripts

Reply:
Need to debug this